code/modules/projectiles/guncrafting/attachments/handles.dm

```diff
@@ -0,0 +1,40 @@
+/obj/item/weapon/gun_attachment/handle/semiauto
+	name = "Semi-Auto Handle"
+	icon_state = "handle_semi"
+	burst_size = 1
+	fire_delay = 1
+	recoil = 1
+	automatic = 0
+	spread = 0
+	random_sprite = list(1,2)
+
+/obj/item/weapon/gun_attachment/handle/burst
+	name = "Burst Handle"
+	icon_state = "handle_burst"
+	burst_size = 2
+	fire_delay = 6
+	recoil = 4
+	automatic = 0
+	spread = 5
+	random_sprite = list(1,3)
+
+/obj/item/weapon/gun_attachment/handle/auto
+	name = "Automatic Handle"
+	icon_state = "handle_auto"
+	burst_size = 1
+	fire_delay = 1
+	recoil = 3
```

Comment:
Minor nitpick. Due to it being fully automatic, I think its recoil should be 5 so it is higher than the recoil of the Burst handle. The Automatic handle is afterall for holding click in and constantly firing.